deno/payloads/v8/_interactions/modalSubmit.ts

```diff
@@ -0,0 +1,17 @@
+import type { APIBaseInteraction, InteractionType, APIModalActionRowComponent } from '../mod.ts';
+
+export interface APIModalSubmission {
+	/**
+	 * A developer-defined identifier for the component, max 100 characters
+	 */
+	custom_id: string;
+	/**
+	 * A list of child components
+	 */
+	components?: APIModalActionRowComponent[];
```

Comment:
Wait, is this what we get? a row of components? That doesn't seem right...how do we get the data from the users 👀